lib/js/components/Form/Checkbox/Checkbox.stories.ts

```diff
@@ -13,36 +18,66 @@ export default {
 } as Meta<typeof Checkbox>;
 
 const StoryTemplate: StoryFn<typeof Checkbox> = (args) => {
-	const [_, updateArgs] = useArgs();
-
 	return {
-		components: { Checkbox },
+		components: { Checkbox, Banner },
 		setup() {
-			return { args };
-		},
-		methods: {
-			onIsSelectedUpdated(isSelected) {
-				updateArgs({
-					isSelected,
-				});
-			},
+			const props = computed(() => {
+				const { default: defaultSlot, ...rest } = args;
+
+				return rest;
+			});
+
+			const defaultSlot = computed(() => args.default);
+
+			return { defaultSlot, props };
 		},
-		template: template('checkbox'),
+		template: `
+			<Checkbox 
+				v-bind="props"
+			>
+				<span v-if="defaultSlot" v-html="defaultSlot" />
```

Comment:
It renders span inside a span which in some edge-cases can be renderef in diffrent way than regular text inside span `__label`. But I didn't find an easy workaround for that if we want to support HTML in the story

Reply:
I'd leave it as is ;)